datatypes/src/collections/feature_collection_builder.rs

```diff
@@ -418,3 +476,166 @@ where
         }
     }
 }
+
+fn estimate_primite_arrow_array_size<T: ArrowPrimitiveType>(builder: &dyn ArrayBuilder) -> usize {
```

Comment:
what s a primite? Do you mean primitive?

Reply:
[f19493f](https://github.com/geo-engine/geoengine/pull/810/commits/f19493f9a55f4fc25cf93bcf5983e0c45cf4d85a)